templates/Dockerfile.j2

```diff
@@ -23,10 +29,8 @@ RUN groupadd -g 1000 elasticsearch && adduser -u 1000 -g 1000 -d /usr/share/elas
 
 WORKDIR /usr/share/elasticsearch
 
-# Download/extract defined ES version. busybox tar can't strip leading dir.
+# Download and extract defined ES version. busybox tar can't strip leading dir.
 RUN wget --progress=bar:force {{ url_root }}/{{ tarball }} && \
-    EXPECTED_SHA=$(wget -O - {{ url_root }}/{{ tarball }}.sha1) && \
```

Comment:
I don't think that's very useful. Real releases will be handled in the release manager, where the file has just been compiled on local disk. There is no chance of network corruption or man-in-the-middle (even with the old the process, the download was HTTPS). Any SHA we check would have just been generated seconds ago on the same processor and the same disk. Verifying it won't add any value.